src/main/java/com/powsybl/openloadflow/network/FirstSlackBusSelector.java

```diff
@@ -7,14 +7,15 @@
 package com.powsybl.openloadflow.network;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public class FirstSlackBusSelector implements SlackBusSelector {
 
     @Override
-    public SelectedSlackBus select(List<LfBus> buses) {
-        return new SelectedSlackBus(buses.get(0), "First bus");
+    public SelectedSlackBus select(List<LfBus> buses, int limit) {
+        return new SelectedSlackBus(buses.stream().limit(limit).collect(Collectors.toList()), "First bus");
```

Comment:
Maybe "first bus" is no more appropriate.

Reply:
I think it is ok to let it singular as it is a name of a method.